rales_engine/db/schema.rb

```diff
@@ -10,11 +10,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 20170503040529) do
+=======
 ActiveRecord::Schema.define(version: 20170503050709) do
+>>>>>>> 7848cec71f109f19b9c26acc36a3ac12737d1048
```

Comment:
make sure these merge conflict lines are **not** included in this commit and don't make it into master!